src/hello_registry.erl

```diff
@@ -116,17 +116,17 @@ handle_call(_Call, _From, State) ->
     {reply, {error, unknown_call}, State}.
 
 handle_info({'EXIT', From, Reason}, Table) ->
-    ?LOG_ERROR("Hello registry received 'EXIT' signal from monitored process '~p' with reason '~p'.", [From, Reason],
-              [{hello_error_reason, {process_exit, Reason}}], ?LOGID51),
+    ?LOG_ERROR("Hello registry : Received 'EXIT' signal from monitored process ~p with reason ~p.", [From, Reason],
+               [{hello_error_reason, {process_exit, Reason}}], ?LOGID51),
     {noreply, Table};
 handle_info({'DOWN', _MRef, process, Pid, Reason}, Table) ->
     Objects = ets:match(Table, {'$1', Pid, '_', '_'}),
-    ?LOG_ERROR("Hello registry received 'DOWN' signal from monitored process '~p' with reason '~p'. Going to clean up associated processes '~p'.", 
-              [Pid, Reason, Objects], [{hello_error_reason, {process_down, Objects, Reason}}], ?LOGID52),
+    ?LOG_ERROR("Hello registry : Received 'DOWN' signal from monitored process ~p with reason ~p. Going to clean up associated processes ~p.", 
```

Comment:
a little nit-picky here: you should decide on whether you want to capitalize after colons or not